tasks.js

```diff
@@ -12,41 +13,72 @@ module.exports = (resources) => {
     let output = {}
 
     try {
-      const data = await setup(await validate( await load(input)))
-      // business logic goes here...
-      // output = ...
-    } catch (e) { 
-      // if (e.message === ...
-      //   throw new Error('...
-      // }
-      // Default case
-      _debug(e.stack)
-      throw (e)
+      const data = await setup(await validate(await load(input)))
+      output = data.files + data.errored_files
+      //upload file to S3
+        // s3_driver.add(shared.PATH, output).then(response => {
+        //   console.log(shared.OK, response);
+        // })
+        //   .catch(error => {
+        //     console.error(shared.ERROR, error);
+        // });
+    } catch (e) {
+      throw new Error(JSON.stringify({
+        status: shared.ERROR,
+        messageError: e.message,
+      }))
     }
 
     return output
 
-    async function load(input={}) {
+    async function load(input = {}) {
       const config = {}
-      // config.var1 = input.var1
-      // ...
-      // config.config1 = await bundled_config.config('CONFIG1')
-      // ...
+      // assing imput object to config
+      config.files = input.Files
+      config.scanned_files = input.scanned_files
+      config.errored_files = input.errored_files
       return config
     }
-    
+
     async function setup(config) {
-      const data = {...config}
-      // ...
+      const data = { ...config }
+      // validated no ids repeate in scanned
+      data.files = await validationIfProcessed(data.files, data.scanned_files)
+      // validate length files elements to push in errors
+      for (let file of data.files) {
+        if (file.length < 15 || file.length > 19 || await alphanumericValidation(file) == true) {
```

Comment:
JavaScript uses triple '=' to compare exact type and value. If we use only '==' then it will only check if the value matches but not the type, so `true` and `'true'` would be considered equal
```suggestion
        if (file.length < 15 || file.length > 19 || await alphanumericValidation(file) === true) {
```